source/blog/2016-05-19-introducing-subteams.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Introducing Subteams
+author: Tom Dale
+tags: Recent Posts
+---
+
+I'm very happy to share with you our plan to scale Ember's day-to-day
+decision-making and coordination–something we're calling Subteams.
```

Comment:
the dash in `coordination–something` might need surrounding spaces or an em-dash